src/sentry/api/serializers/models/repository_project_path_config.py

```diff
@@ -24,4 +24,5 @@ def serialize(self, obj, attrs, user):
             "stackRoot": obj.stack_root,
             "sourceRoot": obj.source_root,
             "defaultBranch": obj.default_branch,
+            "automaticallyGenerated": obj.automatically_generated,
```

Comment:
This enables determining if the code mapping was auto generated within `update_tags`.